GOVERNANCE.md

```diff
@@ -79,9 +79,9 @@ a standard TC motion.
 Changes to TC membership should be posted in the agenda, and may be
 suggested as any other agenda item (see "TC Meetings" below).
 
-No more than 1/3 of the TC members may be affiliated with the same
+No more than 1/4 of the TC members may be affiliated with the same
```

Comment:
I must have miss-counted the current voting members, I thought we were up to 8. We originally wanted 1/4 but moved it down because we didn't have enough voting members.